File: Bars.GkhGJI/libs/B4/model/Dict/erknm/DirectoryERKNM.js
Ext.define('B4.model.dict.erknm.DirectoryERKNM', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'DirectoryERKNM'
    },
    fields: [
        { name: 'Id'},
        { name: 'Name'},
        { name: 'Code'},
        { name: 'CodeERKNM'},
        { name: 'EntityName'}
    ]
});

File: Bars.GkhGJI/Entities/Dict/DirectoryERKNM.cs
namespace Bars.GkhGji.Entities.Dict
{
    using Bars.Gkh.Entities;

    /// <summary>
    /// Справочники ЕРКНМ
    /// </summary>
    public class DirectoryERKNM : BaseGkhEntity
    {
        public virtual string Name { get; set; }

        public virtual string Code { get; set; }

        public virtual string CodeERKNM { get; set; }

        public virtual string EntityName { get; set; }
    }
}